### ISSUE TITLE
Smooth animation is used when `editor.cursorSmoothCaretAnimation` is set to off

### ISSUE THREAD

**Issue by IcySlurpee** [microsoft/vscode#169332]
Does this issue occur when all extensions are disabled?: Yes

- VS Code and OS Version: 
```
Version: 1.75.0-insider (user setup)
Commit: 44531ffc7b7477dbe82750040d8ed69bae2bb140
Date: 2022-12-15T08:33:44.005Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19045
Sandboxed: Yes
---
Version: 1.75.0-insider (Universal)
Commit: 44531ffc7b7477dbe82750040d8ed69bae2bb140
Date: 2022-12-15T08:33:05.575Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin x64 22.2.0
Sandboxed: Yes
```

When `editor.cursorSmoothCaretAnimation` is set to `"off"`, smooth animation is still used. Setting to `"explicit"` does function as intended.

Steps to Reproduce:

1. Open any file or create a new file.
2. Ensure `editor.cursorSmoothCaretAnimation` is set to off.
3. Type or move the cursor with mouse or arrow keys.  Caret moves smoothly when it should not.

**Comment by IcySlurpee** [comment]
As of the recently released insiders build on commit edc432e, this issue is resolved and `editor.cursorSmoothCaretAnimation` works as expected.
```
Version: 1.75.0-insider (user setup)
Commit: edc432e920c3ec2c2af5e8a99b8e4b782633d298
Date: 2022-12-16T05:23:13.187Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19045
Sandboxed: Yes
```

**Comment by alexdima** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/169171
